src/explore-education-statistics-frontend/redirects.js

```diff
@@ -0,0 +1,2262 @@
+const seoRedirects = [
+  {
+    from: '/nbsp;andnbsp;https://www.gov.uk/government/collections/abortion-statistics-for-england-and-wales',
+    to: '/',
+  },
+  { from: '/!nd-statistics/permanent-and-', to: '/' },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/7763f949-eff9-4571-93b4-08db2bd4eec9',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/9348e5bc-c09d-43b4-de90-08d90ae3d820',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/a095bcaa-7c7c-4a99-de75-08d90ae3d820',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/cd50d521-0d75-498e-143b-08dac7b96540',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/d8ecf739-f299-413f-126f-08dacbba9031',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/ef57d23a-f281-4b6b-a55e-08da6007d7ab',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/fcfa509c-8572-4b61-ca3a-08dbcefb484c',
+    to: '/',
+  },
+  {
+    from: '/api/methodologies/%7BmethodologyId%7D/images/feeffb4e-f022-469a-6e23-08dacc6823fb',
+    to: '/',
+  },
+  {
+    from: '/api/releases/%7BreleaseId%7D/images/0eb3fc80-77ad-4d66-8090-a6c2509d50c5',
+    to: '/',
+  },
+  {
+    from: '/api/releases/%7BreleaseId%7D/images/26f6ac12-25d0-4831-5293-08db35b5456a',
+    to: '/',
+  },
+  {
+    from: '/api/releases/%7BreleaseId%7D/images/fdb52a67-704d-44f3-9721-752000ff9ab9',
+    to: '/',
+  },
+  { from: '/data', to: '/' },
+  { from: '/data-', to: '/' },
+
+  { from: '/data-table', to: '/' },
+  {
+    from: '/data-tablesnbsp;-nbsp;springnbsp;2022',
+    to: '/',
+  },
+  { from: '/data-tables/fast-track/[dataBlockId]', to: '/' },
+  { from: '/data-tables/fast-track/[dataBlockParentId]', to: '/' },
+  {
+    from: '/data-tables/fast-track/0164bb4a-6777-4d8f-a1a0-34ce69f75153',
```

Comment:
Have we investigated why these fast tracks are broken? The couple I've looked at are 500ing.

Reply:
Good spot - I hadn't realised that. The ones I tried were all 404s so I assumed just tables that no longer exist, or incorrect guids. I wonder if we could run them all through a tool to assess which are 404s and which are 500s...

Reply:
I wrote a Postman script to call every redirect link and assess the result. From this: 
* Many yield 404s as expected and should indeed be redirected
* Some yield 500s as you've noticed, which will be dealt with in a new ticket: https://dfedigital.atlassian.net/browse/EES-4978 
* Some result in infinite redirect loops, and will also be addressed in a new ticket: https://dfedigital.atlassian.net/browse/EES-4977